release/gcb/gcb_lib.sh

```diff
@@ -96,15 +96,12 @@ function make_istio() {
   cp        "${ISTIO_OUT}/istio-sidecar.deb"   "${OUTPUT_PATH}/deb/"
   cp        "${ISTIO_OUT}"/archive/istio-*z*   "${OUTPUT_PATH}/"
 
-  declare -a ISTIOCTL_ARCHIVES
-  
-  mapfile -t ISTIOCTL_ARCHIVES < <(ls "${ISTIO_OUT}"/archive/istioctl*.tar.gz)
-  mapfile -t ISTIOCTL_ARCHIVES < <(ls "${ISTIO_OUT}"/archive/istioctl*.zip)
-
-  for i in "${ISTIOCTL_ARCHIVES[@]}"; do
-    sha256sum "$i" > "$i.sha256"
+  for file in "${ISTIO_OUT}"/archive/istioctl*.tar.gz; do
+    sha256sum "${file}" > "$file.sha256"
+  done
+  for file in "${ISTIO_OUT}"/archive/istioctl*.zip; do
```

Comment:
why not just `istioctl*` and one loop? lgtm though

Reply:
changed to one loop